kernel/thinkpad.txt

```diff
@@ -0,0 +1,28 @@
+Kernel configuration for IBM ThinkPad
+________________________________________________________________________________
+
+
+X230
+________________________________________________________________________________
+
+A defconfig is sufficent for the machine to boot. Sound and WIFI must be manu-
+ally enabled. A firmware blob for INTEL Centrino Advanced-N 6205 is necessary.
+Thinkpad specific ACPI options _can_ be enabled.
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make defconfig                                                           |
+|   $ cp iwlwifi-6000g2a-6.ucode /usr/lib/firmware                             |
```

Comment:
We should mention where this file comes from.

Reply:
I have mentioned in the text above. I decided against a reference to e.g 'Install KISS'  to where to obain the blobs, because then there would be a reference for every machine which requires those firmware.

Reply:
That's true. I'll think about how to best proceed here. :+1:

Reply:
The reference on where to obtain the firmware would fit nice in kernel/firmware. Then a note to kernel/firmware at the top of articles of this kind would be sufficient.